src/ros_commo.py

```diff
@@ -444,10 +445,8 @@ def __init__(self):
 			self.language_perceived_text_cb)
 
 		# Chat infrastructure text.
-		# from chatbot.msg import ChatMessage
-		#rospy.Subscriber("/robot/speech", chatbot/ChatMessage,
-		#rospy.Subscriber("chatbot_speech", chatbot/ChatMessage,
-		#	self.chat_perceived_text_cb)
+		rospy.Subscriber("chatbot_speech", ChatMessage,
```

Comment:
Instead of doing this, which breaks existing code, could we instead just fix the chatbot to not use the ChatMessage?  If that cannot be fixed right away, then can we do this in some other branch of HR code, so that existing code will continue to work?